HumanCellAtlas/metadata-schema
# Add pull approve functionality to repo. Fixes #866.

### Release notes

- No schema changes.
- Added .pullapprove.yml to top level directory. Fixes #866.

### Reviews requested

- Need 2 Reviewers to approve because this is a new feature
- Last Reviewer to review/approve, merge changes by **Wednesday June 16**

### To test the new feature
1. Go to any open PR (or make a dummy PR).
1. Under the "Files changed" tab, click "Review changes".
1. In the comment section, type `/pullapprove-test`
1. Under this line, copy/paste the entire contents of the .pullapprove.yml  file inside a code block
![Screen Shot 2019-06-05 at 10 19 54](https://user-images.githubusercontent.com/10993501/58945404-b80b8280-877b-11e9-8c8f-c76a358a5b60.png)
1. Click "Submit review"
1. The text will appear as a comment on the PR, followed by an automatically generated comment from the pullapprove bot. Click the bot comment to see the results of what the pullapprove feature would do if it was enabled.
